docs/operations.rst

```diff
@@ -265,7 +216,6 @@ All of the following will raise an :obj:`IndexError`, like in Numpy 1.13 and lat
    z[3, 6]
    z[1, 4, 8]
    z[-6]
-   z[[True, True, False, True], 3, 4]
 
 .. note:: Numpy advanced indexing is currently not supported.
 
```

Comment:
below here: maybe note that `stack` and `concatenate` require matching fill values, and that some operations (e.g., `tensordot`) require a fill value of zero?

Reply:
Done!